pkg/tests/support/helpers.py

```diff
@@ -243,20 +242,21 @@ def __attrs_post_init__(self):
             }
         else:
             self.binary_paths = {
-                "salt": [str(self.run_root)],
-                "api": [str(self.run_root), "api"],
-                "call": [str(self.run_root), "call"],
-                "cloud": [str(self.run_root), "cloud"],
-                "cp": [str(self.run_root), "cp"],
-                "key": [str(self.run_root), "key"],
-                "master": [str(self.run_root), "master"],
-                "minion": [str(self.run_root), "minion"],
-                "proxy": [str(self.run_root), "proxy"],
-                "run": [str(self.run_root), "run"],
-                "ssh": [str(self.run_root), "ssh"],
-                "syndic": [str(self.run_root), "syndic"],
-                "spm": [str(self.run_root), "spm"],
-                "pip": [str(self.run_root), "pip"],
+                "salt": [self.install_dir / "salt"],
```

Comment:
Why is this needing to be updated? I would assume we want it to call the cli script locations, For example, being able to just call `salt-call` instead of `/path/to/salt-call/` 

I'm worried if we change this it will break upgrades as well.

Reply:
I would second that, need wrappers or equivalent placed in regular path, similar to how used now